src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -268,4 +268,47 @@ void f_constraint(
     const tnsr::iaa<DataType, SpatialDim, Frame>&
         three_index_constraint) noexcept;
 // @}
+
+// @{
+/*!
+ * \brief Computes the generalized-harmonic constraint energy normalization.
+ *
+ * \details Computes the generalized-harmonic constraint energy normalization
+ * [the integrand of Eq. (70) of \cite Lindblom:2005qh, with
+ * \f$\Lambda = 1\f$,
+ * \f$m^{ab} = \delta^{ab}\f$, \f$m^{cd} = \delta^{cd}\f$, and
+ * each term scaled by an arbitrary coefficient],
+ * \f{eqnarray}{
+ * N_E & = & g^{ij} \left(K_\psi \partial_i \psi_{ac} \partial_j \psi_{ac}
+ * + K_\Pi \partial_i \Pi_{ac} \partial_j \Pi_{ac}
+ * + K_\Phi g^{kl} \partial_i \Phi_{kac} \partial_j \Phi_{lac}\right),
+ * \f}
+ * \f$\psi_{ab}\f$ is the spacetime metric,
+ * \f$\Pi_{ab}=-t^c\partial_c \psi_{ab}\f$,
+ * \f$\Phi_{iab} = \partial_i\psi_{ab}\f$,
+ * \f$g^{ij}\f$ is the inverse spatial metric, and
+ * \f$K_\psi\f$, \f$K_\Pi\f$, and \f$K_\Phi\f$ are constant multipliers
+ * for each term that each default to 1.0. Note that in this equation,
+ * spacetime indices \f$a,c\f$ are raised and lowered with the
+ * Kronecker delta.
+ */
+template <size_t SpatialDim, typename Frame, typename DataType>
```

Comment:
Is there any good reason to have the multipliers?  If so please document why I would not choose them to be 1

Reply:
To compare with spec, which has multipliers

Reply:
I don't think that is a good reason.

Reply:
I'm fine taking them out then